Dockerfile

```diff
@@ -332,6 +332,9 @@ RUN apk add --no-cache \
         supervisor \
         tar \
         wget \
+        man \
+        man-pages \
```

Comment:
This is not needed.

Reply:
And to explain better, it adds 20MB to the image. :)

Reply:
Removed